src/main/java/com/celements/metatag/MetaTagApi.java

```diff
@@ -0,0 +1,94 @@
+package com.celements.metatag;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+
+import com.celements.metatag.enums.ECharset;
+import com.celements.metatag.enums.EHttpEquiv;
+import com.celements.metatag.enums.ENameNonStandard;
+import com.celements.metatag.enums.ENameStandard;
+import com.celements.metatag.enums.EReferrer;
+import com.celements.metatag.enums.ERobot;
+import com.celements.metatag.enums.EViewport;
+import com.celements.metatag.enums.opengraph.EOpenGraph;
+import com.celements.metatag.enums.twitter.ETwitter;
+import com.celements.metatag.enums.twitter.ETwitterCardType;
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableMap;
+
+public final class MetaTagApi {
```

Comment:
Interressting idea but not discussed and decided upon. PR is the wrong please to bring up new codeing style ideas. Please take this discussion offline of this PR and stick to the style guide /naming convention in use.